guides/V1_GetActionContext.md

```diff
@@ -0,0 +1,86 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: 'get-action-context'
+is: 'experimental'
+---
+
+
+# Get action context information 
+
+If you have registered a Client action handler callback for your App, then when your App is called through this callback, 
+it will receive an action handler context token. Your App needs to use this token with this API to extract the proper 
+action context.
+
+### Path
+```
+GET /v1/apps/{appId}/getActionContext/{actionHandlerContextToken}
+```
+**actionHandlerContextToken** has only 5 minutes lifetime, and can only be used once. It will be deleted once it is used. 
+
+#### Parameters
+
+|Type|Name|Description|Schema|
+|---|---|---|---|
+|**Header**|**Authorization** <br>_required_ |Authorization header, in form of Bearer {access_token}. The access_token is issued as result of an oauth flow by Watson Work Services|String|
+|**Path**|**appId** <br>_required_ |Unique id of an app|String|
+|**Path**|**actionHandlerContextToken** <br>_required_|token generated by caller before|String|
+
+#### Responses
+
+|HTTP Code|Description|Schema|
+|---|---|---|
+|**200**|The action context data is returned and deleted successfully.|[Action Output Entity](#action-output-entity)|
+|**401**|Unauthorized.||
+|**403**|Forbidden.||
+|**404**|Not Found. The app with the given `appId` or the action context data for the given `actionToken` could not be found.||
+|**500**|Internal server error.||
+
+#### Produces
+
+* `application/json`
+
+<a name="action-output-entity"></a>
+
+#### Action Output Entity
+
+The result from this call provides the following payload.
+
+|Name|Description|Schema|
+|---|---|---|
+|**type**  <br>*required*|The action type for the provided context.|string|
+|**appId**  <br>*optional*|Id of the app for this **actionContext**.|string|
+|**userId**  <br>*required*|Id of the user who triggered this action.|string|
+|**actioncontext**  <br>*required*|The action context structure according to the action type.|[ActionContext](#actioncontext)|
```

Comment:
@jarusso  currently there is no actioncontext field for getActionContext api, but spaceId or teamId instead, need update here and also the below description

Reply:
@weihuaw731 perhaps best to show the actual output of the API for each of the 2 actionTypes we have now.  The idea is to describe that you will always get appId, userId for all actionTypes and then the rest that you get varies based on the type of action.

@jarusso maybe there is a way to describe actionContext not as a field to eliminate confusion...  best thing is to put the actual output of the actual API though.